specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/2023-10-01/appconfiguration.json

```diff
@@ -0,0 +1,2103 @@
+              "Sync-Token": {
+                "description": "Enables real-time consistency between requests by providing the returned value in the next request made to the server.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetSnapshots": {
+            "$ref": "./examples/GetSnapshots.json"
+          }
+        },
+        "x-ms-pageable": {
+          "itemName": "items",
+          "nextLinkName": "@nextLink"
+        }
+      },
+      "head": {
+        "tags": [
+          "Snapshots"
+        ],
+        "summary": "Requests the headers and status of the given resource.",
+        "operationId": "CheckSnapshots",
+        "consumes": [],
+        "produces": [],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SyncTokens"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "After",
+            "in": "query",
+            "description": "Instructs the server to return elements that appear after the element referred to by the specified token.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "headers": {
+              "Sync-Token": {
+                "description": "Enables real-time consistency between requests by providing the returned value in the next request made to the server.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response."
+          }
+        },
+        "x-ms-examples": {
+          "CheckSnapshots": {
+            "$ref": "./examples/CheckSnapshots.json"
+          }
+        }
+      }
+    },
+    "/snapshots/{name}": {
+      "get": {
+        "tags": [
+          "Snapshots"
+        ],
+        "summary": "Gets a single key-value snapshot.",
+        "operationId": "GetSnapshot",
+        "consumes": [],
+        "produces": [
+          "application/vnd.microsoft.appconfig.snapshot+json",
+          "application/problem+json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SyncTokens"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "The name of the key-value snapshot to retrieve.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "description": "Used to perform an operation only if the targeted resource's etag matches the value provided.",
```

Comment:
Does getting a single snapshot possible to use a match condition?

Reply:
Yes